filters/no_porn_tokenizer.py

```diff
@@ -0,0 +1,84 @@
+from sacremoses import MosesTokenizer
+from toolwrapper import ToolWrapper
+from subprocess import run, PIPE
+import logging
+import sys
+import os
+import re
+
+
+# variables used by the no_escaping function
+replacements = {"&amp;":  "&",
+                "&#124;": "|",
+                "&lt;":   "<",
+                "&gt;":   ">",
+                "&apos":  "'",
+                "&quot;": '"',
+                "&#91;":  "[",
+                "&#93;":  "]"}
+
+substrs = sorted(replacements, key=len, reverse=True)
+nrregexp = re.compile('|'.join(map(re.escape, substrs)))
+
+# Back-replacements of strings mischanged by the Moses tokenizer
+def no_escaping(text):
+    global nrregexp, replacements
```

Comment:
`global` is not needed here. It would only be needed if you'd be assigning new values to them.